super.c

```diff
@@ -1044,6 +1045,13 @@ static int apfs_check_features(struct super_block *sb)
 		return -EINVAL;
 	}
 
+	features = le64_to_cpu(vsb_raw->apfs_fs_flags);
+	if (!(features & APFS_FS_UNENCRYPTED)){
```

Comment:
```suggestion
	if (!(features & APFS_FS_UNENCRYPTED)) {
```
Add a space here for consistency.